ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -509,6 +520,41 @@ export default class ConfirmApproveContent extends Component {
     return titleTokenDescription;
   }
 
+  renderTitle() {
+    const { t } = this.context;
+    const { isSetApproveForAll, setApproveForAllArg } = this.props;
+    const titleTokenDescription = this.getTitleTokenDescription();
+
+    let title;
```

Comment:
This could be `let title = t('allowSpendToken', [titleTokenDescription]);`, and we avoid the `||` in the return statement, but I won't block on that.

Reply:
haha sorry @adonesky1 and @darkwing. I actually liked the `||` to possibly do one less translation here. I can switch the preference to @darkwing 's for future implementations